### ISSUE TITLE
styling translations

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#4840]
**styling translations**

The sub-header/summary of portal pages are styled with italics, however, this is not used in e.g. Japanese and Chinese.

Any thoughts on how we might get around such conflicts in a general way?

-----

Github user: @dnoesgaard
User: [See in registry](https://www.gbif.org/api/feedback/user/42b683dd2ef0b8c32357c46a2cae3e78:414da9d10ca99575ffa8f448f864c91c9b2d1e645aaf9182ef8253e37b206caab7a343316139e1830a26a2b9339645923fad7a549e6e3db6faf6d68c1d9af9a57a42afcd708e74f402b13d161bdcbd5bbc666609ddf06bb8e37b29d108e23d63e3964730926851e7dc34314ba6d42aecfd3b88be7129f3b6128a617262449bdd4dd1179599871bfd6c9cddcf1d23e862) - [Send email](https://www.gbif.org/api/feedback/user/mailto/42b683dd2ef0b8c32357c46a2cae3e78:414da9d10ca99575ffa8f448f864c91c9b2d1e645aaf9182ef8253e37b206caab7a343316139e1830a26a2b9339645923fad7a549e6e3db6faf6d68c1d9af9a57a42afcd708e74f402b13d161bdcbd5bbc666609ddf06bb8e37b29d108e23d63e3964730926851e7dc34314ba6d42aecfd3b88be7129f3b6128a617262449bdd4dd1179599871bfd6c9cddcf1d23e862)
System: Chrome 114.0.0 / Mac OS X 10.15.7
Referer: https://www.gbif.org/news/5WZThcL928vmPnSvrGhZfE/report-reveals-return-on-investments-in-gbif
Window size: width 1701 - height 1248
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-06-28T07:26:11.136Z',to:'2023-06-28T07:32:11.136Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-06-28T07:26:11.136Z',to:'2023-06-28T07:32:11.136Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by MattBlissett** [comment]
CSS can vary by language:

```
  h3:lang(zh), h3:lang(ja) {
    font-weight: bold;
  }

  em:lang(zh), em:lang(ja) {
    font-style: normal;
    text-emphasis: dot;
  }
```

(But for a heading, bold would probably be more appropriate.)

**Comment by MortenHofft** [comment]
Yes we can add custom styling per language. Weird that it even can be italized if it doesn't make sense. I would have thought the browser/font took care of that.

**Comment by dnoesgaard** [comment]
I agree, it's weird. I leave it to you to decide what's the best way to handle this. We might have similar issues with other languages/scripts...
